Added Lap Divider and ability to switch between graphs

KensonN opened the pull request.

abdullahelqaq commented:
Hmm are you pushing changes from a fork? I'm sure it's fine if you're the only one committing to the repo, but it's not really the workflow we're after. @ashvinnagarajan thoughts?

ashvinnagarajan commented on src/App.css:
```diff
@@ -154,4 +154,8 @@ html {
+
+/* option:checked {
+  display: none;
+} */
```
whats the point of this commented code. Just remove this if unnecessary

KensonN replied:
Leftover from testing, removed!

ashvinnagarajan reviewed (commented):
> Hmm are you pushing changes from a fork? I'm sure it's fine if you're the only one committing to the repo, but it's not really the workflow we're after. @ashvinnagarajan thoughts?

Definitely preferable to work off a clone of the repo if possible. So much easier if you have multiple people editing the same files

ashvinnagarajan commented:
I started looking at this PR but there is just so much itll take a long time. @KensonN Can you please help out by writing a summary of the changes and also including some images of the UI that will be added/changed?

ashvinnagarajan commented:
I completely trust that this code works beautifully. As Abs has mentioned to others in the past, the point of these PRs is really so that we can all fully understand what's going on.

KensonN commented:
Summary
----------
- Added dropdown menu to switch graphs between different data.
- Added a visual lap divider when the lap button is pressed on the ground dash.
- Removed WhichOne component and moved functionality into function called ViewToggle directly in App.js.
- Moved periodic check for when the script is running into it's own function.

To-do this sprint
- Reduce lag by cutting down on looping used to retrieve data for the graphs.
- Implement chart data streaming plugin to have graphs move horizontally with the data as data becomes harder to see as time goes on with current graphs.

KensonN commented:
Can you clarify on working on a clone of the repo? What does that exactly entail?

ashvinnagarajan commented:
> Can you clarify on working on a clone of the repo? What does that exactly entail?

Great question!

ashvinnagarajan reviewed (approved):
Looks good!

KensonN commented:
Gotcha, I'll work on getting a clone of the repo made

ashvinnagarajan commented:
> Gotcha, I'll work on getting a clone of the repo made

Yeah no worries! Next time!

anaymehta reviewed (approved):
Looks excellent, can't wait to see this on the website!
